### DISCUSSION TITLE
Unused event images

### DISCUSSION THREAD

**Discussion by rnyoakum** [raceintospace/raceintospace#631] [General]
I was looking through src/game/news.cpp, specifically `void ShowEvt(char plr, char crd)`, which displays the image corresponding to the news event during the broadcast. It contains a section

```
    // Apparently not all of the news events are associated with the
    // right images? This would suggest some images get used twice.
    if (plr == 0) {
        switch (crd) {
        case 41:
            crd = 45;
            break;

        case 50:
            crd = 67;
            break;

        case 8:
            crd = 98;
            break;

        case 105:
            crd = 112;
            break;

        case 33:
            crd = 111;
            break;

        case 109:
            crd = 108;
            break;

        default:
            break;
        }
    }
```

For some reason, the corresponding images for 6 of the USA events are ignored and others substituted (and doing double-duty). The card images exist in data/images/news/, so I'm not sure why they aren't being used. Does anyone know the reason for this?

Back when the news images were collected in a single file, hard coding a fix may have been a reasonable alternative to modifying the file. But the event images are now stored as individual pngs. The correct approach is to ensure the right image resources are in place and properly named.

**Comment by peyre** [comment]
Sounds to me like something we should fix.  We might as well use all the images!
